docs/decisions/0007-reducing-complexity.md

```diff
@@ -0,0 +1,39 @@
+# Reducing Complexity
+
+## Motivation
+
+High complexity is in general a concerning problem: It can lead to untested legacy code, a high amount of bugs, bad maintainability, slow development, and so on. These decisions are concrete changes we want to make to reduce complexity.
+
+## Decisions
+
+### Hooks
+
+Decision: Hooks always start with `use` (for example `useCounter`). This applies to custom hooks. A custom hook is any function that calls other hooks, especially state and lifecycle hooks like `useEffect` or `useState`. Functions that do not call other hooks never start with `use`.
+
+Reason: See general principles (3). See [React Docs](https://beta.reactjs.org/learn/reusing-logic-with-custom-hooks#hook-names-always-start-with-use)
+
+### Contents of hooks.js
+
+Decision: A hooks.js file lives in the directory of a React component, and must contain only custom hooks. All other functions that are not custom hooks must be in other files. (You may make an exception for functions that you consider private to that hooks.js file and do not expect to be used anywhere else.) All custom hooks that belong to the parent component must be in that hooks.js file. This does not apply for hooks that are intended to be reusable.
+
+Reason: It is common practice to have a "hooks.js" file and you will only expect hooks in there. We put all (except for reused) hooks in there in order to make it easy to find hooks and distinguish between them and normal functions.
+
+### Reusable hooks
+
+Decision: Hooks intended to be reusable live in shared directories. Either the directory name or the filename they are in must signify that they are hooks.
```

Comment:
Design or coding question?

Reply:
Just for throwing out ideas where to put stuff if we don't have all our functions in the hooks file. Maybe we should not have this in here as it's not an important point?

Reply:
I removed this point